Stp.TestingApi/Controllers/TaskController.cs

```diff
@@ -32,21 +32,30 @@ public IEnumerable<Contracts.TaskDto> GetTasksByCategory(long taskCategoryId)
                 .Include(x => x.MultichoiceAnswers)
                 .Select(x => new Contracts.TaskDto()
```

Comment:
Use just class name without namespace (move "Contracts" part to usings). Here and in other similar places.